packages/client/typings/window.d.ts

```diff
@@ -70,7 +70,6 @@ interface Window {
     DATE_OF_BIRTH_UNKNOWN: boolean
     INFORMANT_SIGNATURE: boolean
     USER_NOTIFICATION_DELIVERY_METHOD: string
-    INFORMANT_NOTIFICATION_DELIVERY_METHOD: string
```

Comment:
@Nil20 Let's not remove the type from here as the endpoint does actually return this value right?

Reply:
So it was a duplicate one